File: Algoritmo(C)/Alg_S8_4.c
#include <stdio.h>
int main(){
    //variables
    int vetor[20], i, soma = 0;
    //input
    for(i = 0; i <= 19; i++){
        printf("Digite o valor do vetor %d/20: ",(i + 1));
        scanf("%d",&vetor[i]);
        //process
        soma += vetor[i];
    }
    //output
    printf("A soma e: %d",soma);
}

File: Algoritmo(Python)/Alg_S8_4.py
#variables
vetor = []
#soma = 0
#input + process
for(i) in range(0, 20):
    num = int(input("Digite um número para o vetor {0}/20: ".format(i + 1)))
    vetor.append(num)
    #soma += num
print(sum(vetor)) #soma